doc: Add Server Address mismatch error when using TTS CLI (#1309)
TheThingsIndustries/lorawan-stack-docs: doc/content/the-things-stack/interact/cli/troubleshooting/_index.md

```bash
ttn-lw-cli gateways set <gateway-id> --location-public=false --status-public=false --auto-update=true
```

## Server Address Mismatch

> Accessing {{% tts %}} CLI requires the correct [CLI configuration file]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) with the server addresses. If you try to perform actions on the entities registered in other server address, then you will face the error `error:cmd/ttn-lw-cli/commands:end_device_server_address_mismatch`.
> 
> For example, if you configured the CLI with the `eu1` cluster server address and try to access the device registered on the `au1` cluster, then the below error will be observed due to the mismatch of server addresses in the [CLI configuration file]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}).

Reviewer: ```suggestion
If you [configure {{% tts %}} CLI]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) to use one cluster server address (for example `eu1`) and try to perform actions on the entities registered in other cluster (for example `au1`), you will probably face the following error:
```

```
error:cmd/ttn-lw-cli/commands:end_device_server_address_mismatch (Network/Application/Join Server address mismatch)
```

> While configuring the server address in the [CLI configuration file]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}), you need to ensure that the correct cluster address is used for performing any operations on entities registered in that cluster. If you want to perform operations on the entities registered in the `eu1` cluster then you need to ensure the [CLI configuration file]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}) with the `eu1` cluster.

Reviewer: ```suggestion
If you want to perform operations on the entities registered in a certain cluster, then you need to ensure the CLI is configured with server addresses for that cluster.
```

> It is important to use the `eu1` cluster as the Identity Server address in the [CLI configuration file]({{< ref "/the-things-stack/interact/cli/configuring-cli" >}}), regardless of the entity registered in any other cluster.

Reviewer: ```suggestion
Keep in mind that Identity Server and OAuth addresses are always in `eu1` cluster, as described [here]({{< ref "/the-things-stack/cloud/addresses#command-line-interface" >}}).
```
